tlatools/org.lamport.tlatools/src/tla2sany/drivers/SANY.java

```diff
@@ -77,6 +77,10 @@
     // true <=> statistics about builtin operator usage 
     //          should be reported
 
+  private static boolean strictErrorCodes   = false;
```

Comment:
For consistency with the rest: Please rename to `doStrictErrorCodes`

Reply:
Done